### ISSUE TITLE
SFTP  > 2GB

### ISSUE THREAD

**Issue by MerzkayaGnida** [Magisk-Modules-Repo/ssh#13]
When copying (SFTP) files of large size (> 2GB), the size of the copy file is limited to 2GB without an error message ...

**Comment by eMPee584** [comment]
Yeah I'm having similar issues on an Xperia X Compact on the stock ROM.. might be because of some _battery-optimizing_ background process killing?
What kind of ROM are you on?

**Comment by MerzkayaGnida** [comment]
Probably the reason is 32-bit scp ...
